src/components/event/EventCardHeader.tsx

```diff
@@ -83,8 +84,18 @@ function EventCardHeader({
 	};
 
 	return (
-		<div className={rootClassName} onClick={onSelect} style={rootStyle}>
-			<div className={iconClassName}></div>
+		<div
+			className={rootClassName}
+			onClick={onSelect}
+			style={rootStyle}
+			onMouseEnter={() => {
+				eventStore.setHoveredEvent(event);
```

Comment:
we definitely need some debouncing for this, otherwise it will be called to ofter and often unnecessary